### PR TITLE
Update limit-guest-sharing-to-specific-organization.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the impact on existing guests and teams' chats.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10516

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/solutions/limit-guest-sharing-to-specific-organization.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If you only want your users to invite guests from specific organizations, you ca
 The first step in limiting guest sharing is to change the default settings in the Azure AD cross-tenant access settings to block inviting guests by default. Then you can allow guest invitations for specific organizations.
 
 > [!NOTE]
-> Changes to cross-tenant access settings may take two hours to take effect.
+> Changes to cross-tenant access settings may take two hours to take effect. Changes will block access for existing guests whose domains are not added to cross-tenant access organizational settings. Changes will also block all communication inbound via teams with any domain not configured in the organization settings.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/limit-guest-sharing-to-specific-organization.md]:
```suggestion
> Changes to cross-tenant access settings may take two hours to take effect. Changes will block access for existing guests whose domains are not added to cross-tenant access organizational settings. Changes will also block all communication inbound via Teams with any domain not configured in the organizational settings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT not merged yet, please review. Thanks

**Pull request merged**